src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -281,6 +281,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <!-- Example, c:\MyProjects\MyProject\bin\debug\MyAssembly.dll -->
     <TargetPath Condition=" '$(TargetPath)' == '' ">$(TargetDir)$(TargetFileName)</TargetPath>
 
+    <TargetRefPath Condition=" '$(TargetRefPath)' == '' and '$(Deterministic)' == 'true' ">$([System.IO.Path]::Combine(`$([System.IO.Path]::GetDirectoryName($([System.IO.Path]::GetFullPath(`$(TargetPath)`))))`, 'ref', `$(TargetFileName)`))</TargetRefPath>
```

Comment:
I think if a user wants to set $(TargetPath) they should also have to set $(TargetRefPath) right?  There's no guarantee that appending `ref\$(TargetFileName)` to whatever they specified as `TargetPath` would make sense.  I'll let you decide...